spinalcordtoolbox/scripts/sct_analyze_texture.py

```diff
@@ -206,7 +206,12 @@ def mean_angle(self):
             # Average across angles and save it as wrk_folder/fnameIn_feature_distance_mean.extension
             fname_out = im_m + str(self.param_glcm.distance) + '_mean' + extension
             run_proc('sct_image -i ' + ' '.join(im2mean_lst) + ' -concat t -o ' + fname_out)
```

Comment:
Just to make things even clearer, you could add a comment here saying something like:

```
            # We specify '-concat t' here just to concatenate along the 4th dimension 
            # But, to be clear, the 4th dimension in this case is "angle" and not "time"
```

Admittedly It's a bit weird that we have to specify `t` for these functions instead of just specifying a dimension (0/1/2/3). :stuck_out_tongue:

Reply:
Actually, can we replace this call to `sct_image` entirely, while we're at it?

I mean, we have `image.concat_data`, right? See also this call, which accomplishes the same thing, I think (concatenating across the 4th dimension):

https://github.com/neuropoly/spinalcordtoolbox/blob/6129ec7ccb16b777462d57276e1f63498a1f9ef2/spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py#L198

Reply:
Yes absolutely.